autoPyTorch/datasets/base_dataset.py

```diff
@@ -13,36 +13,36 @@
 
 from autoPyTorch.constants import CLASSIFICATION_OUTPUTS, STRING_TO_OUTPUT_TYPES
 from autoPyTorch.datasets.resampling_strategy import (
-    CROSS_VAL_FN,
+    CrossValFuncs,
     CrossValTypes,
+    CrossValFunc,
     DEFAULT_RESAMPLING_PARAMETERS,
-    HOLDOUT_FN,
     HoldoutValTypes,
-    get_cross_validators,
-    get_holdout_validators,
-    is_stratified,
+    HoldOutFuncs,
+    HoldOutFunc
 )
 from autoPyTorch.utils.common import FitRequirement, hash_array_or_matrix
 
-BASE_DATASET_INPUT = Union[Tuple[np.ndarray, np.ndarray], Dataset]
+BaseDatasetType = Union[Tuple[np.ndarray, np.ndarray], Dataset]
```

Comment:
maybe we can have this called BaseDatasetInputType

Reply:
Ah, ok good to hear. I will change it to this name then.